apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/src/main/java/gov/cms/bfd/pipeline/rda/grpc/apps/LoadRdaJsonApp.java

```diff
@@ -126,10 +152,21 @@ private Config(ConfigLoader options) {
       mcsFile = options.readableFileOption("file.mcs");
     }
 
+    /**
+     * Sets database options for db Url, user, password, and the max poolsize.
```

Comment:
More flexible to not enumerate all the parameters used (think about maintainability.) Strive for a description that doesn't bind _too_ tightly to the implementation. (Just "Creates the {@link DatabaseOptions} from this configuration." is probably enough for this one.)